### PR TITLE
4.25.0 changelog

### PR DESCRIPTION
**Pull request by pmaytak**

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,11 +1,37 @@
+4.25.0
+==========
+### New Features: 
+**MSAL.NET now advertises PKAuth support only on supported platforms**. See issues [#1849](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1849), [2302](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2302) for details.
```
Comment (jennyf19) [inline changelog.txt:4]:
```suggestion
**MSAL.NET now advertises PKAuth support only on supported platforms**. See issues [#1849](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1849), [#2302](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2302) for details.  
```

File: changelog.txt [hunk 2]
```diff
@@ -1,11 +1,37 @@
+4.25.0
+==========
+### New Features: 
+**MSAL.NET now advertises PKAuth support only on supported platforms**. See issues [#1849](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1849), [2302](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2302) for details.  
+**`WithPreferredAzureRegion` allows specifying an option to fallback to global endpoint if the region lookup fails**. See issue [#2287](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2287) for details.
```
Comment (jennyf19) [inline changelog.txt:5]: we have an "Experimental Features" section now, I would move the regional items there, and I guess WAM as well.

File: changelog.txt [hunk 3]
```diff
@@ -1,11 +1,37 @@
...
+**`WithPreferredAzureRegion` allows specifying an option to fallback to global endpoint if the region lookup fails**. See issue [#2287](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2287) for details.  
+**`WithPreferredAzureRegion` allows specifying a region to use**. See issue [#2259](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2259) for details.  
+**Optimized IMDS calling logic during regional lookup**. See issue [#2177](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2177) for details.  
+**WAM is enabled on net5.0-windows10.0.17763.0**. See issue [#2274](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2274) for details.  
+**Added support for embedded view  for .NET 5.0 projects**. See issue [#2310](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2310) for details.  
+**Improved handling of broker's power optimization exception in Xamarin**. See issue [#2144](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2144) for details.
```
Comment (jennyf19) [inline changelog.txt:10]:
```suggestion
**Improved handling of broker's power optimization exception in Xamarin Android**. See issue [#2144](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2144) for details.  
```

File: changelog.txt [hunk 4]
```diff
@@ -1,11 +1,37 @@
...
+**`WithClientAssertion` allows specifying a delegate to set the assertions**. See issue [#2184](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2184) for details.  
+
+### Bug Fixes:
+**Regional lookup now correctly uses a global endpoint when `WithPreferredAzureRegion` is set to `false` after the initial lookup was done with a regional endpoint**. See issue [#2260](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2260) for details.  
+**WAM can now be used in console apps**. See issue [#2196](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2196) for details.  
+**WAM support is moved to a separate package, `Microsoft.Identity.Client.Desktop`**, which fixes dependency issues during build. See issues [#2299](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2299), [#2300](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2300), [2247](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2247) for details.
```
Comment (jennyf19) [inline changelog.txt:17]:
```suggestion
**WAM support is moved to a separate package, `Microsoft.Identity.Client.Desktop`**, which fixes dependency issues during build. See issues [#2299](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2299), [#2300](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2300), [#2247](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2247) for details.  
```

File: changelog.txt [hunk 5]
```diff
@@ -1,11 +1,37 @@
...
+**In WAM, account picker is now correctly not used when `AcquireTokenInteractive` is called with the default OS account**. See issue [#2246](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2246) for details.  
+**Account is not longer deleted from the MSAL cache when a `bad_token` response is received from the authentication server**. See issue [#2294](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2294) for details.  
+**Minimum UWP target supported is now 10.0.0.0**. See issue [#2330](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2330) for details.  
+**Fixed `ArgumentNullException` and improved resiliency when using `RSACryptoServiceProvider`**. See issue [#2189](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2189) for details.  
+**Honoring a provided redirect URI in `WithRedirectUri`**. See issue [#2167](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2167) for details.
```
Comment (jennyf19) [inline changelog.txt:22]: this is specifically for when using a localhost with a path. might want to be more specific in the description.

**Review by jennyf19 — APPROVED** [review]

File: changelog.txt (hunk 1 above)
Comment (pmaytak) [reply changelog.txt]: Resolved.

File: changelog.txt (hunk 3 above)
Comment (pmaytak) [reply changelog.txt]: Resolved.

File: changelog.txt (hunk 4 above)
Comment (pmaytak) [reply changelog.txt]: Resolved.

File: changelog.txt (hunk 5 above)
Comment (pmaytak) [reply changelog.txt]: Resolved.

File: changelog.txt (hunk 2 above)
Comment (pmaytak) [reply changelog.txt]: Resolved.

**Review by bgavrilMS — APPROVED** [review]

**Review by neha-bhargava — APPROVED** [review]

**Pull request merged**